addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5286,7 +5286,12 @@ msgctxt "#12373"
 msgid "Settings & file manager"
 msgstr ""
 
-#empty strings from id 12374 to 12375
+#empty strings with id 12374
+
+#: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
+msgctxt "#12375"
+msgid "This will save selected values as default for all media. Changes to 'View mode' or 'Pixel ratio' affect aspect ratio on playback. Are you sure?"
```

Comment:
@da-anda Done

![screenshot025](https://cloud.githubusercontent.com/assets/3521959/10764013/21c96e88-7cc4-11e5-8f82-a534966a2b0d.png)

Not sure if thats a stern enough warning, but its not specific like you said.

let me know